### ISSUE TITLE
Keine Daten mehr vom Wechselrichter

### ISSUE THREAD

**Issue by DJFS** [tobiasfaust/SolaxModbusGateway#24]
Hallo,  
nach einen kompletten neuflashen des ESP32 lief es ca. 10 Tage ohne Probleme, Jedoch bekomme ich jetzt seit 3 Tagen keine Werte mehr vom Wechselrichter per Modbus.
Weboberfläche und MQTT gehen. 

_

_```
<html>
<body>

Typ | Raw Data
-- | --
RawData of ID-Data |  
RawData of Live-Data | undefined undefined undefined undefined undefined undefined undefined undefined undefinedundefined undefined undefined undefined undefined undefined undefined undefined undefined

</body>
</html>

_
```_

**Comment by tobiasfaust** [comment]
Restarten und beobachten….
Ein Log (Level 5) zum Zeitpunkt des Abrisses wäre hilfreich.
Konnte ich bisher auch nicht nachvollziehen

**Comment by DJFS** [comment]
ein Reset hat nichts gebracht,  evtl neu flashen?
woher bekomme ich den log?

Die Daten von der letzten MQTT übertragen kann ich anbieten:

![image](https://github.com/tobiasfaust/SolaxModbusGateway/assets/134227035/28eafdf4-2460-40af-acfd-8fd71bb183e3)

**Comment by tobiasfaust** [comment]
Log bekommt man über die serielle Konsole. Also ESP32 an den PC anschließen und über den seriellen Port das Log (Level 4 oder 5) beobachten

**Comment by DJFS** [comment]
![image](https://github.com/tobiasfaust/SolaxModbusGateway/assets/134227035/302939da-8761-46e0-b21c-0adcc7b4ab22)


Es ging mehrere Tage ohne Probleme und am 11.07. 07:57:08 war Schluss.

**Comment by DJFS** [comment]
Hab nun den ESP neu geflasht,
der AP "AP_Solax"  hatte ein Passwort?  Kenne aber keins

**Comment by tobiasfaust** [comment]
Sorry, Passwort im WIKI hinzugefügt. Das Passwort ist in einem der letzen changes hinzugekommen.

https://github.com/tobiasfaust/SolaxModbusGateway/wiki/start-and-integration-into-locale-wifi

**Comment by tobiasfaust** [comment]
Laut log sieht es so aus als das du items aktiviert hattest, aber vergessen zu speichern. Deswegen lief es am Anfang.
Nach einem reboot sind keine items mehr aktiviert weil die Config nicht gespeichert wurde. Einfach mal auf der weboberfläche wieder aktivieren und speichern nicht vergessen.

**Comment by DJFS** [comment]
Ich habe bisher nichts weiter gemacht. Ich bekomme es nicht zum laufen. Vllt hole ich mir einfach den original Stick. Oder ich schau mal nach Tasmota o.ä.

**Comment by tobiasfaust** [comment]
Und du hast komplett von vorne nochmal angefangen? Mit dem aktuellen Repo stand aus dem Development Branch?

**Comment by lexxmm** [comment]
klasse Ergebnisse hier...habe genau das gleiche Problem...von jetzt auf gleich keine Daten mehr...

**Comment by tobiasfaust** [comment]
Etwas mehr Details wären sicher hilfreich.

**Comment by lexxmm** [comment]
Genau wie bei DJFS...von heute auf morgen keine Daten mehr. Nur undefined undefined undefined undefined undefined undefined undefined undefined undefined
undefined undefined undefined undefined undefined undefined undefined undefined undefined

log daten: 

Ajax Json Antwort: 
Query Live Data into Queue:
0x01 0x04 0x04 0x00 0x00 0x40 
Request queue data to inverter: 0x01 0x04 0x04 0x00 0x00 0x40 0xF0 0xCA 
Read Data from Queue: 
no response from client
Parse 0 Bytes of data
Ajax Json Empfangen: {
  "action": "RefreshLiveData"
}

WR ist ein X3-MIC-4K-G2  

Es wurde NICHTS geändert. Habe stundenlang alles probiert und echt keine Lust mehr :(

**Comment by lexxmm** [comment]
genau..kein Update. Ausschalten (gewisse Zeit) von beiden Geräten habe ich schon versucht...hat nix gemacht. Also ist der WR defekt?

**Comment by tobiasfaust** [comment]
Ich kann mir nicht vorstellen das der WR kaputt gegangen ist.  Vielleicht haben sich die Modbus Einstellungen des WR verstellt?
Da kann ich nicht wirklich helfen ….

**Comment by DJFS** [comment]
Hallo lexxmm,

ich habe den esp einmal komplett gelöscht und neu geflasht,  nach den ic dann alle Einstellungen wieder drin hatte lief er wieder

**Comment by tobiasfaust** [comment]
Das wäre dann schon sehr „strange“ ;)
Meiner läuft seit genau „ 164 Days, 18 Hours, 51 Minutes“ durch ohne jegliche Probleme…..

**Comment by lexxmm** [comment]
ok scheinbar wirklich WR defekt...jetzt arbeitet er nämlich gar nicht mehr. Nach ein paar Sekunden kommen komische Zeichen auf dem Display und er startet neu.
